Explorer/Assets/DCL/Ipfs/SceneMetadata.cs

```diff
@@ -50,5 +58,57 @@ public struct Coordinate
                 public float[]? MultiValue { get; internal set; }
             }
         }
+
+        /// <summary>
+        /// Configuration specific to Decentraland Worlds (Realms).
+        /// {
+        ///     "worldConfiguration": {
+        ///     "name": "my-name.dcl.eth",
+        ///     "skyboxConfig": {
+        ///         "fixedTime": 36000
+        ///     },
+        ///     "fixedAdapter": "offline:offline"
+        /// }
+        /// 
+        /// skyboxConfig.fixedTime: This property indicates how many
+        /// seconds have passed (in Decentraland time) since the start of the day,
+        /// assuming the full cycle lasts 24 hours. Divide the seconds value by 60 to obtain minutes,
+        /// and by 60 again to obtain the hours since the start of the day.
+        /// For example, if the seconds value is 36000, it corresponds to 10:00 am.
+        /// If no value is set for this field, the world will follow the same day/night cycle as Genesis City.
+        /// 
+        /// NOTE: more about the setup: https://docs.decentraland.org/creator/worlds/about/
+        /// </summary>
+        [Serializable]
+        public class WorldConfiguration
+        {
+            /// <summary>
+            /// The unique name of the world (e.g., an ENS name).
+            /// </summary>
+            public string? Name { get; set; }
+
+            /// <summary>
+            /// Configuration for the world's skybox.
+            /// </summary>
+            public SkyboxConfig? SkyboxConfig { get; set; }
+            
+            /// <summary>
+            /// Specifies the adapter used for scene deployment or loading (e.g., "offline:offline").
+            /// </summary>
+            public string? FixedAdapter { get; set; }
+        }
+
+        /// <summary>
+        /// Defines settings for the world's skybox.
+        /// </summary>
+        [Serializable]
+        public class SkyboxConfig
```

Comment:
Same, could this be a struct? We avoid unnecessary allocations, and we are just holding data here